build/yamls/antrea-windows.yml

```diff
@@ -96,6 +96,11 @@ data:
     # hybrid:            noEncap if source and destination Nodes are on the same subnet, otherwise encap.
     #
     #trafficEncapMode: encap
+
+    # Provide the name of the interface on Node which is used for the Pod traffic across Nodes. If there are multiple
+    # IP addresses configured on the interface, a random one is chosen. It is always the first one.
```

Comment:
So if not set, it will be the random/first/NodeIP?

Reply:
If not set, NodeIP is used. If the configuration is set, in the meanwhile multiple IPs are configured on the specified interface, the first addre is used.